components/config/src/config/mod.rs

```diff
@@ -100,16 +137,10 @@ pub struct Config {
 
     /// The setup for which slugification strategies to use for paths, taxonomies and anchors
     pub slugify: slugify::Slugify,
-
-    /// The search config, telling what to include in the search index
-    pub search: search::Search,
-
-    /// All user params set in [extra] in the config
-    pub extra: HashMap<String, Toml>,
 }
```

Comment:
I don't know about that..
Things like `compile_sass`, `minify_html` etc are not language specific. Maybe pass something like `language_config` to the template instead?

Reply:
Yep. I mainly want to have language-specific search because CJK search indices can get huge; and tight bandwidth might exclude things like feeds